examples/validate_relative_binding.py

```diff
@@ -0,0 +1,347 @@
+# This script runs the rabfe code in "inference" mode.
```

Comment:
Is relative binding another term for rabfe?

Reply:
This is really an absolute method